src/compas_fab/robots/robot.py

```diff
@@ -31,21 +31,24 @@ class Robot(Data):
     upon the model described in the class :class:`compas_robots.RobotModel` of
     the **COMPAS** framework.
 
-    Attributes
+    Parameters
     ----------
-    model : :class:`RobotModel`
+    model : :class:`compas_robots.RobotModel`
```

Comment:
You don't need to specify the package if it's already imported. Unless you're doing this intentionally to highlight that the model is coming from another library?

Reply:
Yea
 That was the intention, see earlier reply